assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx

```diff
@@ -585,6 +585,15 @@ const ExternalActivity: React.FC<PartComponentProps<CapiIframeModel>> = (props)
             typeof changedVar.value === 'number'
           ) {
             changedVar.type = CapiVariableTypes.NUMBER;
+          } else if (
+            changedVar.type === CapiVariableTypes.STRING &&
```

Comment:
so, the EverApp was sending value of 'app.spr-app-adaptivity.SW.Where.iNumPlanetsFound'  = "0" i.e. as a string value and there is a condition in trap state 

`let app.spr-app-adaptivity.SW.Where.iNumPlanetsFound = {app.spr-app-adaptivity.SW.Where.iNumPlanetsFound}+1`

and it was failing during evaluation in scripting.